src/ray/object_manager/pull_manager.h

```diff
@@ -196,26 +191,94 @@ class PullManager {
     // object. This includes bundle requests whose objects are not actively
     // being pulled.
     absl::flat_hash_set<uint64_t> bundle_request_ids;
+
+    bool IsPullable() const { return object_size_set && !pending_object_creation; }
   };
 
-  struct PullBundleRequest {
-    PullBundleRequest(const std::vector<rpc::ObjectReference> &requested_objects)
-        : objects(requested_objects), num_object_sizes_missing(objects.size()) {}
-    const std::vector<rpc::ObjectReference> objects;
-    size_t num_object_sizes_missing;
-    // The total number of bytes needed by this pull bundle request. Note that
-    // the objects may overlap with another request, so the actual amount of
-    // memory needed to activate this request may be less than this amount.
-    size_t num_bytes_needed = 0;
-
-    void RegisterObjectSize(size_t object_size) {
-      RAY_CHECK(num_object_sizes_missing > 0);
-      num_object_sizes_missing--;
-      num_bytes_needed += object_size;
+  /// A helper structure for tracking information about each ongoing bundle pull request.
+  struct BundlePullRequest {
+    BundlePullRequest(std::vector<ObjectID> requested_objects,
+                      BundlePriority bundle_priority)
+        : objects(std::move(requested_objects)), priority(bundle_priority) {}
+    const std::vector<ObjectID> objects;
+    const BundlePriority priority;
```

Comment:
Is this field used? Wondering why we need it now but didn't previously.

Reply:
oops, it's a left over from my previous implementation.